app/src/main/scala/org/alephium/explorer/persistence/model/TokenInfoEntity.scala

```diff
@@ -16,10 +16,12 @@
 
 package org.alephium.explorer.persistence.model
 
+import org.alephium.explorer.api.model.StdInterfaceId
 import org.alephium.protocol.model.TokenId
 import org.alephium.util.TimeStamp
 
 final case class TokenInfoEntity(
     token: TokenId,
-    lastUsed: TimeStamp
+    lastUsed: TimeStamp,
```

Comment:
Yes, everytime we insert block data, we check for tokens and insert them in the token_info table with the following constraint:
```
INSERT .....
.....
ON CONFLICT
DO UPDATE
SET last_used = EXCLUDED.last_used
```
This allows latter to order tokens by this last_used field. To avoid having a forgotten token on the top of the page when we request tokens' info.

Reply:
We should update this field at most once per day/week. The order should be pretty stable in the future. The most popular tokens would be the few ones.